workers/generic-worker/garbagecollector.go

```diff
@@ -7,19 +7,19 @@ import "fmt"
 // favour of a resource with a lower rating.
 type Resource interface {
 	Rating() float64
-	Expunge(task *TaskRun) error
+	Evict(task *TaskRun) error
```

Comment:
@petemoore and I talked last week about the strange naming and that `Evict()` is more fitting here.